src/App/App.js

```diff
@@ -1,10 +1,31 @@
-import './App.css';
-import HomePage from '../HomePage/HomePage'
+import React, { Component } from "react";
+import "./App.css";
+import HomePage from "../HomePage/HomePage";
+import MovieSnapShot from "../MovieSnapShot/MovieSnapShot";
 
-function App() {
-  return (
-   <HomePage/>
-  );
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      clickedMovieId: ""
+    }
+  }
+
+  onMovieClick = (id) => {
+    this.setState({ clickedMovieId: id })
+  }
+  onHomeClick = () => {
+    this.setState({ clickedMovieId: "" })
+  }
+
+  render() {
+    return (
+      <>
+        {!this.state.clickedMovieId ?
+        <HomePage onMovieClick={this.onMovieClick}/> : <MovieSnapShot onHomeClick={this.onHomeClick}/>}
+      </>
+    )
```

Comment:
Could think about using a variable here for the conditional render for readability: 

const userView = !this.state.clickedMovieId ? `<HomePage onMovieClick={this.onMovieClick}/>` : `<MovieSnapShot onHomeClick={this.onHomeClick}/>`

return (
<>
  {userView}
</>
)

Reply:
When refactoring this weekend we can make that change if that the direction we want to go.